bundle-workflow/src/run_integ_test.py

```diff
@@ -16,56 +15,19 @@
 from manifests.test_manifest import TestManifest
 from system import console
 from system.temporary_directory import TemporaryDirectory
+from test_workflow.dependency_installer import DependencyInstaller
 from test_workflow.integ_test.integ_test_suite import IntegTestSuite
 from test_workflow.test_args import TestArgs
 
-# TODO: 1. log test related logging into a log file. Output only workflow logs on shell.
-# TODO: 2. Move common functions to utils.py
 
-COMMON_DEPENDENCIES = ["OpenSearch", "common-utils", "job-scheduler", "alerting"]
-
-
-# TODO: replace with DependencyProvider - https://github.com/opensearch-project/opensearch-build/issues/283
-def pull_common_dependencies(work_dir, build_manifest):
-    logging.info("Pulling common dependencies for integration tests")
+def pull_build_repo(work_dir):
     logging.info("Pulling opensearch-build")
     os.chdir(work_dir)
     GitRepository(
         "https://github.com/opensearch-project/opensearch-build.git",
         "main",
         os.path.join(work_dir, "opensearch-build"),
     )
-    for component in build_manifest.components:
-        if component.name in COMMON_DEPENDENCIES:
-            logging.info("Pulling " + component.name)
-            GitRepository(
-                component.repository,
-                component.commit_id,
-                os.path.join(work_dir, 'dependencies', component.name),
-            )
-
-
-# TODO: replace with DependencyProvider - https://github.com/opensearch-project/opensearch-build/issues/283
-def sync_dependencies_to_maven_local(work_dir, manifest_build_ver):
-    dependencies_dir = os.path.join(work_dir, 'dependencies')
-    opensearch_dir = os.path.join(dependencies_dir, 'OpenSearch')
-    os.chdir(opensearch_dir)
-    deps_script = os.path.join(
-        work_dir,
-        "opensearch-build/tools/standard-test/integtest_dependencies_opensearch.sh",
```

Comment:
todo: `integtest_dependencies_opensearch.sh` file can be removed now

Reply:
any reason not to delete it part of this PR?

Reply:
i realized that after putting up the PR, will do now :)